src/main/java/seedu/address/model/person/PersonComparator.java

```diff
@@ -12,7 +13,7 @@ public class PersonComparator {
     public static final String NAME = "name";
     public static final String DATE_OF_LAST_VISIT = "date of last visit";
     public static final String EARLIEST_VALID_DATE = "01-01-0001";
-    public static final String LATEST_VALID_DATE = "31-12-9999";
+    public static final String LATEST_VALID_DATE = LocalDate.now().format(DateOfLastVisit.DATE_TIME_FORMATTER);
```

Comment:
It may be better to not change this. The LATEST_VALID_DATE is meant to distinguish entries with no date of last visit from those that do have. I believe in this case if you set the date of last visit to be today for one of the entries and then sort the entry with dolv as today will be mixed with those without date of last visit. Hence, it is better to keep the value as is or to set it as today + 1.

Reply:
DateOfLastVisit has been updated to accept a date no later than today, which means that a date of "31-12-9999" would not be possible. I have updated the comparators for date of last visit to not use EARLIEST_VALID_DATE and LATEST_VALID_DATE. Although this makes the code slightly longer and less clean, the functionality works as expected.